### DISCUSSION TITLE
Running the CLI

### DISCUSSION THREAD

**Discussion by turibe** [crowbarz/aiopioneer#30] [Q&A] [answered]
A very basic question: how can I run the CLI from the unix prompt? `test_api.py` has `if __name__ == "__main__":`, but `cli.py` does not. (This seems to be missing from the README.) Thx!

**Comment by crowbarz** [comment, marked as answer]
This is covered in [Command line interface](https://github.com/crowbarz/aiopioneer#command-line-interface-cli--013-cli-arguments--03). Once you have installed the package, then `aiopioneer` should start the CLI.

**Reply by turibe** [reply]
Thanks! Perhaps what can be clarified is that you need to do the install first (`python setup.py install`), and then you will find the `aiopioneer` executable in the `bin` directory for your python env.

**Reply by crowbarz** [reply]
Note that the primary purpose of this package is to support the related Home Assistant integration for Pioneer AVRs, and is thus normally installed as part of that integration from Home Assistant. The CLI is provided as a convenience for troubleshooting only. It is not designed to be used standalone as an interface to an AVR: use either the Home Assistant integration, or the (currently undocumented) Python API that is provided by this package.
